### PR TITLE
Add helm version

### PR DESCRIPTION
**Pull request by brendarearden**
Add helm version to determine if app was deployed by helm 2 or helm 3

### CODE REVIEW

File: apis/project.cattle.io/v3/app_types.go [hunk 1]
```diff
@@ -29,6 +29,7 @@ type AppSpec struct {
 	Prune               bool              `json:"prune,omitempty"`
 	MultiClusterAppName string            `json:"multiClusterAppName,omitempty" norman:"type=reference[/v3/schemas/multiclusterapp]"`
 	ValuesYaml          string            `json:"valuesYaml,omitempty"`
+	HelmVersion         string            `json:"helmVersion,omitempty"`
```
Comment (mrajashree) [inline apis/project.cattle.io/v3/app_types.go]: do we need nocreate/noupdate tag?

Comment (brendarearden) [reply apis/project.cattle.io/v3/app_types.go]: yes I believe so  - I will add that! 😄

File: apis/management.cattle.io/v3/multi_cluster_app.go [hunk 2]
```diff
@@ -35,6 +35,7 @@ type MultiClusterAppSpec struct {
 	Roles                []string        `json:"roles,omitempty" norman:"type=array[reference[roleTemplate]],required"`
 	RevisionHistoryLimit int             `json:"revisionHistoryLimit,omitempty" norman:"default=10"`
 	UpgradeStrategy      UpgradeStrategy `json:"upgradeStrategy,omitempty"`
+	HelmVersion          string          `json:"helmVersion,omitempty" norman:"nocreate,noupdate"`
```
Comment (dramich) [inline apis/management.cattle.io/v3/multi_cluster_app.go]: Let's move this to status instead. Spec indicates something that a user sets while status is controlled internally. Also in the future status will be protected similar to how k8s does it but through 'steve'.

File: apis/project.cattle.io/v3/app_types.go [hunk 3]
```diff
@@ -29,6 +29,7 @@ type AppSpec struct {
 	Prune               bool              `json:"prune,omitempty"`
 	MultiClusterAppName string            `json:"multiClusterAppName,omitempty" norman:"type=reference[/v3/schemas/multiclusterapp]"`
 	ValuesYaml          string            `json:"valuesYaml,omitempty"`
+	HelmVersion         string            `json:"helmVersion,omitempty" norman:"nocreate,noupdate"`
```
Comment (dramich) [inline apis/project.cattle.io/v3/app_types.go]: Let's move this to status instead. Spec indicates something that a user sets while status is controlled internally. Also in the future status will be protected similar to how k8s does it but through 'steve'.

**Pull request merged**